### PR TITLE
New permissions

### PR DESCRIPTION
**Pull request by ericgaspar**
## Solution
- *New permissions*
- *Small linter warnings*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
*If you have access to [App Continuous Integration for packagers](https://yunohost.org/#/packaging_apps_ci) you can provide a link to the package_check results like below, replacing '-NUM-' in this link by the PR number and USERNAME by your username on the ci-apps-dev. Or you provide a screenshot or a pastebin of the results*

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/onlyoffice_ynh%20PR-NUM-%20(USERNAME)/)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -21,7 +21,7 @@ path_url=$(ynh_app_setting_get --app=$app --key=path)
 is_public=$(ynh_app_setting_get --app=$app --key=is_public)
 db_name=$(ynh_app_setting_get --app=$app --key=db_name)
 db_user=$db_name
-db_pwd=$(ynh_app_setting_get --app=$app --key=db_pwd)
+db_pwd=$(ynh_app_setting_get --app=$app --key=psqlpwd)
```
Comment (alexAubin) [inline scripts/upgrade:24]: Gotta handle the migration for this setting ... if in the past it was called db_pwd, gotta move it to psqlpwd in the upgrade script

**Review by alexAubin — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**